CMakeLists.txt

```
@@ -66,12 +70,20 @@ endif()
 
 if(NOT RYML_DEFAULT_CALLBACKS)
     target_compile_definitions(ryml PRIVATE RYML_NO_DEFAULT_CALLBACKS)
+else()
+    if(RYML_DEFAULT_CALLBACK_USES_EXCEPTIONS)
+        target_compile_definitions(ryml PRIVATE RYML_DEFAULT_CALLBACK_USES_EXCEPTIONS)
+    endif()
 endif()
 
 if(RYML_DBG)
     target_compile_definitions(ryml PRIVATE RYML_DBG)
 endif()
 
+if(RYML_USE_ASSERT)
+    target_compile_definitions(ryml PUBLIC RYML_USE_ASSERT=1)
```

```suggestion
    target_compile_definitions(ryml PUBLIC RYML_USE_ASSERT)
```
should be sufficient.

---

I did it again! Nice catch, thanks!

---

Actually, no. I misread your suggestion and thought that you were proposing adding `=1` instead of removing it. 

Indeed, both are equivalent. But `=1` is more explicit, and consistent to what the value of the macro should be. I will stick to it.